vite.config.ts

```diff
@@ -8,4 +8,14 @@ export default defineConfig({
   build: {
     outDir: "dist/frontend",
   },
+  server: {
+    // proxy /trpc requests
+    proxy: {
+      "/trpc": {
+        target: "http://localhost:4000", // Your Express server
```

Comment:
in development, i proxy http requests from frontend (localhost:5743 or whatever the frontend vite dev server is) starting with "/trpc" to localhost:4000 (the backend nodejs server).  I do this because the cookie is http only (meaning javascript cannot acccess it) and same domain only (meaning it's send along in all requests to the domain of the same page).